data_steward/deid/config/ids/tables/death.json

```diff
@@ -1,8 +1,16 @@
         }
     ]
```

Comment:
not finding the alias `death` in `death.person_id = record2.person_id and cause_source_concept_id`